kloppy/domain/models/common.py

```diff
@@ -119,6 +119,25 @@ def __str__(self):
         return self.value
 
 
+@dataclass(frozen=True)
+class Referee:
```

Comment:
"Referee" or "Official"? I don't have a strong preference but one reason to prefer "Official" covers (e.g., the fourth official).

Reply:
I wasn't sure here either, but I will update it to Official.